test/cqlpy/test_keyspace.py

```diff
@@ -10,18 +10,14 @@
 from cassandra.protocol import SyntaxException, AlreadyExists, InvalidRequest, ConfigurationException
 from threading import Thread
 
-from test.cluster.util import parse_replication_options
+from test.cluster.util import parse_replication_options, get_replication
```

Comment:
Does this get_replication() also work for Cassandra?
If it doesn't, it can't be used in test/cqlpy (unless it's supposed to be testing a Scylla-only feature).